src/main/java/org/sunbird/portal/SbCbExtApplication.java

```diff
@@ -14,7 +14,6 @@
 import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
 import org.springframework.web.client.RestTemplate;
 
-@EnableJpaRepositories(basePackages = {"org.sunbird"})
```

Comment:
is this caused any other existing APIs ??

Reply:
I have enhanced the DB-specific configuration and targeted the correct package with @EnableJpaRepositories() where this is being used. @EnableJpaRepositories(basePackages = {"org.sunbird"}) This will not impact on any other API because earlier it targeted the root package but now we will only target the specific package where it will be used because we are using two different databases.